src/components/NativeSelect/story.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC, useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import NativeSelect, { SimpleSelectOptionType } from '.';
+import { boolean } from '@storybook/addon-knobs';
+
+const options: Array<SimpleSelectOptionType> = [
+    {
+        value: 'A',
+        label: 'Option A',
+    },
+    {
+        value: 'B',
+        label: 'Option B',
+    },
+    {
+        value: 'C',
+        label: 'Option C',
+    },
+    {
+        value: 'D',
+        label: 'Option D',
+    },
+    {
+        value: 'E',
+        label: 'Option E',
+    },
+    {
+        value: 'F',
+        label: 'Option F',
+    },
+];
+
+type PropsType = {};
+
+const Demo: FC<PropsType> = (): JSX.Element => {
+    const [value, setValue] = useState<SimpleSelectOptionType['value']>('E');
```

Comment:
You don't really have to cast this since it can be inferred.
```suggestion
    const [value, setValue] = useState('E');
```